src/devices/src/virtio/net/persist.rs

```diff
@@ -88,10 +88,13 @@ impl Persist<'_> for Net {
         )
         .map_err(Error::CreateNet)?;
 
-        // Safe to unwrap because MmdsNetworkStack::restore() cannot fail.
-        net.mmds_ns = state.mmds_ns.as_ref().map(|mmds_state| {
-            MmdsNetworkStack::restore(constructor_args.mmds.clone(), &mmds_state).unwrap()
-        });
+        // We trust the MMIODeviceManager::restore to pass us an MMDS data store reference if
+        // there is at least one net device having the MMDS NS present.
```

Comment:
I would also add a check for the other way around: no MMDS namespace, but data store reference is present.

Reply:
> I would also add a check for the other way around: no MMDS namespace, but data store reference is present.

I wouldn't do this, since this is not really an unrecoverable error. It should just be ignored. WDYT?